test/property-list.css

```diff
@@ -4924,6 +4941,14 @@
     padding-left: 0;
     padding-left: 10%;
 
+    padding-inline: 1ch;
+    padding-inline-start: 1ch;
+    padding-inline-end: 1ch;
+
+    padding-block: 1ch;
+    padding-block-start: 1ch;
+    padding-block-end: 1ch;
+
```

Comment:
Thanks for adding these! You'll notice that `property-list.css` is missing the magic comments found in the `syntax_test_*.css` files. These are not automated tests. I use `property-list.css` to visually scan for errors. It's crude, but it catches mistakes. Before `syntax_test`s were invented, that was the best I could do.